Documentation/building/windows/instructions.md

```diff
@@ -20,21 +20,18 @@ MSBuild version 15 or later is required.
 
  5. In a [Developer Command Prompt][developer-prompt], prepare the project:
 
-        dotnet msbuild Xamarin.Android.sln -t:Prepare
+        dotnet msbuild Xamarin.Android.sln -t:Prepare -nr:false -m:1
```

Comment:
I'd also prefer using the "long form" instead of "short form" of parameters, e.g. `-nodeReuse:false` instead of `-nr:false`.

I'm still not sure why we need `-nr:false`, though.  *CI* doesn't appear to be to use that, except within unit tests: https://github.com/xamarin/xamarin-android/blob/c22b2a8bbbfb906d5d9029147dda7cfaf68d947a/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs#L224

Reply:
It stops you getting errors locally when rebuilding the tree. I'll see if I can get the error message I see.

Reply:
This is the message I get then not disabling `nodeReuse`.

```
 error MSB4018: The "GenerateDepsFile" task failed unexpectedly. [/Users/dean/Documents/Sandbox/Xamarin/WI1714603/external/Java.Interop/tools/jcw-gen/jcw-gen.csproj]
       /Users/dean/Documents/Sandbox/Xamarin/WI1714603/bin/Release/dotnet/sdk/8.0.100-rc.1.23373.1/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(267,5): error MSB4018: System.IO.IOException: The process cannot access the file '/Users/dean/Documents/Sandbox/Xamarin/WI1714603/bin/Release/lib/packs/Microsoft.Android.Sdk.Darwin/34.0.0/tools/jcw-gen.deps.json' because it is being used by another process. [/Users/dean/Documents/Sandbox/Xamarin/WI1714603/external/Java.Interop/tools/jcw-gen/jcw-gen.csproj
```